system/security/shared/caching.cpp

```diff
@@ -537,6 +525,21 @@ inline void CPermissionsCache::removeAllManagedFileScopes()
     m_managedFileScopesMap.clear();
 }
 
+static CriticalSection setCacheTimeoutCriticalSection;
+void  CPermissionsCache::setCacheTimeout(unsigned timeoutSeconds)
```

Comment:
Should this function be deleted?  I don't think it is called, and theoretically allowing it to change introduces some race conditions.

Reply:
I agree.